WalletWasabi/Extensions/LinqExtensions.cs

```diff
@@ -39,14 +39,15 @@ public static class LinqExtensions
 		}
 	}
 
-	public static T? RandomElement<T>(this IEnumerable<T> source)
+	public static T? RandomElement<T>(this IEnumerable<T> source, WasabiRandom random)
```

Comment:
Without deep investigation - is it possbile to default to `InsecureRandom.Instance`?

Reply:
ACK.